exporter/exporterhelper/common.go

```diff
@@ -217,3 +218,8 @@ func (ts *timeoutSender) send(req request) error {
 	}
 	return req.export(ctx)
 }
+
+// DefaultUserAgent User-Agent value used by exporters
+func DefaultUserAgent(info component.BuildInfo) string {
+	return fmt.Sprintf("%s/%s", info.Description, info.Version)
```

Comment:
It might be nice to add `GOOS` and `GOARCH` here as well.

Reply:
Updated in the new commit